CHANGELOG.md

```diff
@@ -63,18 +63,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Use postEvent for emitting events safely
-## [0.1.0] - 2021-12-03
+## [0.1.0](https://github.com/KDAB/cxx-qt/releases/tag/v0.1.0) - 2021-12-03
 
 ### Added
 
 - Initial release
 - Support for bridging properties and invokables
 - Support for QColor, QPointF, QSizeF, QString, QVariant
 - Support for nested objects
-
-[Unreleased]: https://github.com/KDAB/cxx-qt/compare/v0.4.0...HEAD
-[0.3.0]: https://github.com/KDAB/cxx-qt/compare/v0.3.0...v0.4.0
```

Comment:
This one was incorrect as the left side had 0.3.0 instead of 0.4.0. 

Move these to be inline as it's easy to forget about this at the bottom and have typos :see_no_evil: